Resources/Private/Fusion/Prototypes/Field.fusion

```diff
@@ -2,18 +2,35 @@ prototype(Neos.Fusion.Form:Field)  < prototype(Neos.Fusion:Component) {
     @context.field = ${Form.createFieldDefinition(this.form, this.name, this.property, this.value)}
 
     @propTypes {
-        @strict = true
```

Comment:
why did you remove strictness? IMHO enabling this leads to better error messages :)

Reply:
strict causes hard to get errors when inherited prototypes add more props